From f717defaef643ed5acf17dd75e0992cee904f575 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Sat, 16 Mar 2019 23:22:29 +0100
Subject: [PATCH 1/2] windowCycler: Create settings before chaining up

It's used in _getWindows() which is called from the parent's _init().

https://gitlab.gnome.org/GNOME/gnome-shell/issues/1064
---
 js/ui/altTab.js | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/js/ui/altTab.js b/js/ui/altTab.js
index cdd5d8e21..fb9c875c8 100644
--- a/js/ui/altTab.js
+++ b/js/ui/altTab.js
@@ -619,9 +619,8 @@ class WindowSwitcherPopup extends SwitcherPopup.SwitcherPopup {
 var WindowCyclerPopup = GObject.registerClass(
 class WindowCyclerPopup extends CyclerPopup {
     _init() {
-        super._init();
-
         this._settings = new Gio.Settings({ schema_id: 'org.gnome.shell.window-switcher' });
+        super._init();
     }
 
     _getWindows() {
-- 
2.21.0

